jetson/odrive_bridge/src/__main__.py

```diff
@@ -66,6 +72,16 @@ def lcmThreaderMan():
     lcm_1.subscribe("/drive_vel_cmd", drive_vel_cmd_callback)
     while True:
         lcm_1.handle()
+        global start_time
+        start_time = t.clock()
+        try:
+            publish_encoder_msg()
+        except NameError:
+            pass
+        except AttributeError:

```

Comment:
same thing, when does this happen?

Reply:
this happens when the odrive is disconnected suddenly and it tires to do odrive.function before going to the disconnected state & upon bootup